clinical-mdr-api/generate_validation_report.py

```
@@ -42,77 +68,144 @@ def generate_report():
-            test_name = sanitize_test_name(scenario_name)
-            status = get_test_status(xml_root, test_name)
+            metadata, body = parse_markdown_metadata(content)
             
-            feature_scenarios.append({
-                'name': scenario_name,
-                'fs_ids': list(set(scenario_fs_ids)),
-                'steps': steps,
-                'status': status
+            iso_type = metadata.get('ISO_TYPE', '')
+            iso_category = get_iso_category(iso_type)
+            
```

Pylint (4.0.5)

[convention] Trailing whitespace

(C0303)